diff --git a/node_modules/@digitalcredentials/jsonld-signatures/lib/suites/LinkedDataSignature.js b/node_modules/@digitalcredentials/jsonld-signatures/lib/suites/LinkedDataSignature.js
index 3aec367..4bb5517 100644
--- a/node_modules/@digitalcredentials/jsonld-signatures/lib/suites/LinkedDataSignature.js
+++ b/node_modules/@digitalcredentials/jsonld-signatures/lib/suites/LinkedDataSignature.js
@@ -204,7 +204,7 @@ module.exports = class LinkedDataSignature extends LinkedDataProof {
       documentLoader,
       // throw errors if any values would be dropped due to missing
       // definitions or relative URLs
-      safe: true,
+      safe: false,
       skipExpansion,
       useNative: this.useNativeCanonize,
       ...this.canonizeOptions
@@ -301,7 +301,7 @@ module.exports = class LinkedDataSignature extends LinkedDataProof {
       '@context': constants.SECURITY_CONTEXT_URL,
       '@embed': '@always',
       id: verificationMethod
-    }, {documentLoader, compactToRelative: false, safe: true});
+    }, {documentLoader, compactToRelative: false, safe: false});
     if(!framed) {
       throw new Error(`Verification method ${verificationMethod} not found.`);
     }
